packages/react-menu/src/components/Menu/Menu.test.tsx

```diff
@@ -451,4 +451,49 @@ describe('Menu', () => {
     // Assert
     getByText(visible);
   });
+
```

Comment:
Perhaps add a test for `inline` as well? Could be done in a future PR.

Reply:
Yeah, I think I just carelessly forgot about that. I added the test